openwisp_users/tests/test_admin.py

```diff
@@ -30,8 +32,6 @@
 User = get_user_model()
 Group = load_model('openwisp_users', 'Group')
 
-devnull = open(os.devnull, 'w')
-
```

Comment:
Was opened outside of class but closed in `TearDown`. Therefore, it may cause issues if another tests are added that try to silence noise.